.devcontainer/devcontainer.json

```diff
@@ -4,7 +4,7 @@
   "name": "Louis Login Frontend Node.js project",
   // Use 'postCreateCommand' to run commands after the container is created.
   "image": "mcr.microsoft.com/devcontainers/javascript-node:0-18",
-  "postCreateCommand": "npm install -g npm@9.8.1 --force",
+  "postCreateCommand": "npm install -g npm@9.8.1 --force && sleep 5 && npm i @saithodev/ts-appversion",
```

Comment:
Why not add the `@saithodev/ts-appversion` dependency to package.json? I'm suspecting the 5 second delay has something to do with it?

Reply:
@k-allagbe the 5 second sleep interval is there to mitigate any possible error when jumping between npm install and ts-appversion install.